packages/flutter_tools/lib/src/commands/build_bundle.dart

```diff
@@ -18,22 +18,9 @@ class BuildBundleCommand extends BuildSubCommand {
     usesTargetOption();
     usesFilesystemOptions(hide: !verboseHelp);
     usesBuildNumberOption();
-    addBuildModeFlags(verboseHelp: verboseHelp);
+    addBuildModeFlags(verboseHelp: verboseHelp, defaultToRelease: false);
     usesExtraDartFlagOptions();
     argParser
```

Comment:
This flag makes no sense now that the re-entrant build is gone. Before it would essentially tell this command to not generate kernel, because kernel generation would be conditionally performed by build aot